Automattic/sensei
# Add onboarding theme step

### Changes proposed in this Pull Request

* [x] Include a new step in the onboarding wizard to present the Course theme.
* [ ] Add a new task to the home to customize the styles if the theme is installed.
* [ ] Install and activate the theme in the last onboarding step if the user click on "Install the new Sensei theme".

The incomplete items will come as part of other PRs to avoid a very complex PR.

### Testing instructions


* Navigate to the Setup Wizard: `/wp-admin/admin.php?page=sensei_setup_wizard`
* Go through the steps, and make sure now it has a theme step.
* Navigate through the theme step in different screen sizes, and make sure it works properly.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/204379967-e2cd78ee-f848-4352-aba2-6a2119d9d0e6.mp4

https://user-images.githubusercontent.com/876340/204378233-5aadcb79-e7e9-41d9-90bc-6656d3dbe115.mov
